webapp/i18n/en.json

```diff
@@ -298,6 +296,8 @@
   "admin.email.smtpUsernameDescription": " Obtain this credential from administrator setting up your email server.",
   "admin.email.smtpUsernameExample": "E.g.: \"admin@yourcompany.com\", \"AKIADTOVBGERKLCBV\"",
   "admin.email.smtpUsernameTitle": "SMTP Server Username:",
+  "admin.email.enableSMTPAuthTitle": "Enable SMTP Authentication:",
+  "admin.email.enableSMTPAuthDesc": "When enabled the username and passsword below are used to authenticate to the SMTP server. Some servers require authentication to be disabled.",
```

Comment:
Proposed help text:

```
When enabled, username and password are used for authenticating to the SMTP server.
```

Also, if you'd like, would you be able to change the help text for the following two settings? (I can also prepare a separate PR if you'd prefer, I just thought it's a good opportunity to improve the help texts, since the settings are related)

```
SMTP Server Username
Help text: The username for authenticating to the SMTP server.

SMTP Server Password
Help text: The password for the SMTP server username.
```